BinaryVDF.test_loads_utf16 not passed

Some tests not passed in Fedora 32:

```
=================================== FAILURES ===================================
__________________________ BinaryVDF.test_loads_utf16 __________________________
self = <tests.test_binary_vdf.BinaryVDF testMethod=test_loads_utf16>
    def test_loads_utf16(self):
>       self.assertEqual({'aaa': b'b\x00b\x00\xff\xffb\x00b\x00'.decode('utf-16le')}, vdf.binary_loads(b'\x05aaa\x00b\x00b\x00\xff\xffb\x00b\x00\x00\x00\x08'))
E       AssertionError: {'aaa': 'bb\uffffbb'} != {'aaa': '戀戀\uffff戀戀'}
E       - {'aaa': 'bb\uffffbb'}
E       ?          ^^      ^^
E       
E       + {'aaa': '戀戀\uffff戀戀'}
E       ?          ^^      ^^
tests/test_binary_vdf.py:188: AssertionError
```

But tests passed in Fedora 33 and no such issue. Full [F32 build log](https://kojipkgs.fedoraproject.org//work/tasks/713/60470713/build.log).

Additional information:

- OS: `Fedora 32`
- Python ver: `3.8.7`

--- a/vdf/__init__.py
+++ b/vdf/__init__.py
@@ -361,7 +361,7 @@ def binary_load(fp, mapper=dict, merge_duplicate_keys=True, alt_format=False, ra
         result = buf[:end]
 
         if wide:
-            result = result.decode('utf-16')
+            result = result.decode('utf-16le')
         elif bytes is not str:
             result = result.decode('utf-8', 'replace')
         else:
@@ -469,7 +469,7 @@ def _binary_dump_gen(obj, level=0, alt_format=False):
                 value = value.encode('utf-8') + BIN_NONE
                 yield BIN_STRING
             except:
-                value = value.encode('utf-16') + BIN_NONE*2
+                value = value.encode('utf-16le') + BIN_NONE*2
                 yield BIN_WIDESTRING
             yield key + BIN_NONE + value
         elif isinstance(value, float):

